test/e2e/e2e_suite_test.go

```diff
@@ -180,6 +179,28 @@ func createEnvoy() {
 	testutils.CreateObjsFromYaml(testConfig, manifests)
 }
 
+func createInferencePool(targets int, toDelete bool) {
```

Comment:
nit:
targets could mean pods or endpoint (ip:port) for the inference pool.
Based on usage below, seems to be neither, but rather a port count for the pool